test/client/getTransactions/index.ts

```diff
@@ -1,41 +1,41 @@
-import {RippleAPI} from 'ripple-api'
+import {Client} from 'xrpl-client'
```

Comment:
xrpl-client is already taken on `npm.js`, I think we'll want this to be either `xrpl` or `xrpl-js`